### ISSUE TITLE
Got an error generating response...

### ISSUE THREAD

**Issue by Darkhub** [jake83741/vnc-lm#2]
Running bot via npm on windows, it gives this error:

  cause: AggregateError [ECONNREFUSED]:
      at internalConnectMultiple (node:net:1118:18)
      at afterConnectMultiple (node:net:1685:7) {
    code: 'ECONNREFUSED',
    [errors]: [ [Error], [Error] ]
  }
}
Error generating response: TypeError: responseStream.on is not a function
    at handleResponseStream (C:\vnc-lm\dist\message-generation\chunk-generation.js:39:20)
    at Client.handleMessageCreate (C:\vnc-lm\dist\message-generation\message-create.js:102:83)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    
    Any idea on how to solve?
    
    Thanks.

**Comment by jake83741** [comment]
I think I’ve received that error in the past when I’ve asked a question and Ollama was not running. Could you make sure Ollama is running?

**Comment by Darkhub** [comment]
I just fixed it, all was up&running, bot started and connected on ollama on another machine, found model and so on.
I was pretty sure that the IP address of the machine running ollama was correctly entered then.

But in the log of npm I found that there was still some 'localhost' referred in ollama api url, at this point i went into source files and hardcoded the IP address instead of the default 'http://localhost:11434' and, guess what? It is working now.

So at this point the question is: how has the IP address of the other machine running ollama server to be entered in .env file?
At the moment it is: OLLAMAURL=http://192.168.1.100:11434
